contracts/gateway/BaseAmplifierGateway.sol

```diff
@@ -224,15 +229,16 @@ abstract contract BaseAmplifierGateway is IBaseAmplifierGateway {
 
     /**
      * @dev For backwards compatibility with `validateContractCall`, `commandId` is used here instead of `messageId`.
+     * @return hash the message hash
      */
     function _messageHash(
         bytes32 commandId,
         string calldata sourceChain,
         string calldata sourceAddress,
         address contractAddress,
         bytes32 payloadHash
-    ) internal pure returns (bytes32) {
-        return keccak256(abi.encode(commandId, sourceChain, sourceAddress, contractAddress, payloadHash));
+    ) internal pure returns (bytes32 hash) {
+        hash = keccak256(abi.encode(commandId, sourceChain, sourceAddress, contractAddress, payloadHash));
```

Comment:
same, doesn't need to be named. although good to have the natspec doc

Reply:
updated.